charts/athens-proxy/templates/NOTES.txt

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.ingress.enabled }}
 {{- range .Values.ingress.hosts }}
-  http://{{ . }}
+  https://{{ . }}
```

Comment:
@rimusz can this be configurable and defaulted to `https`?

Reply:
@rimusz and I discussed offline, we'll make this in a follow up, since it's only in `NOTES.txt`